lib/indexer/popularity_lookup.rb

```diff
@@ -16,37 +16,43 @@ def lookup_popularities(links)
             path_components: links,
           },
         },
-        _source: { includes: %w[rank_14] },
+        _source: { includes: %w[rank_14 vc_14] },
         sort: [
           { rank_14: { order: "asc" } },
         ],
         size: 10 * links.size,
       })
 
-      ranks = Hash.new(traffic_index_size)
+      default_rank = Hash.new(
+        rank: traffic_index_size,
+        view_count: 1,
+      )
 
-      results["hits"]["hits"].each do |hit|
+      ranks = results["hits"]["hits"].each_with_object(default_rank) do |hit, hsh|
         link = hit["_id"]
-        rank = Array(hit["_source"]["rank_14"]).first
+        rank = Array(hit.dig("_source", "rank_14")).first
+        view_count = hit.dig("_source", "vc_14") || 1
         next if rank.nil?
 
-        ranks[link] = [rank, ranks[link]].min
+        hsh[link] = { rank: rank, view_count: view_count }
       end
```

Comment:
Previously, `ranks[link]` would have always returned a value.  Now `ranks[link]` will only return a value if the `link` is the `_id` of some result.  Given that, later on, we don't check if `ranks[link]` is non-`nil` before using it, is this a problem?

Reply:
ranks[link] should still always return a value, since the hash has a default value: `Hash.new(rank: traffic_index_size, view_count: 1)`

Reply:
Introducing a new variable confused me far too much 🤦‍♂